mantle/cmd/plume/release.go

```diff
@@ -171,28 +194,14 @@ func modifyReleaseMetadataIndex(api *aws.API) {
 		plog.Fatalf("unmarshaling release metadata json: %v", err)
 	}
 
+	// XXX: switch the URL to be relative so we don't have to hardcode its final location?
 	releasePath := filepath.Join(prefix, "builds", specVersion, "release.json")
```

Comment:
when I was reviewing this change it almost seemed like you made a mistake because we define a `releasePath` both here and in the new `getReleaseMetadata()` function. Maybe we should factor this out into a function too and then just make the use of `url.String()` below call that function since AFAICT `url` is only used in that one place.

Reply:
A function that just builds the path to the `release.json`?

Reply:
no, sorry.

A function that returns the `url` from the `url, err :=` below.

Reply:
Hmm, I'm not sure if it's worth factoring that out. It'd only be useful for this callsite (it wouldn't be useful for `getReleaseMetadata` since we download using the S3 API there and not via the URL). But cool if someone wants to factor it out anyway in a follow-up! Going to merge this since CI is a delicious green.